packages/manager/modules/sms/src/sms/compose/tips/telecom-sms-sms-compose-tips-compose.html

```diff
@@ -197,7 +205,9 @@
                 <tbody>
                     <tr>
                         <td width="20%">€</td>
-                        <td width="20%" data-uib-tooltip-html-unsafe="{{ 'sms_compose_tip_size_tooltip_characters_label' | translate }} {{ 'sms_compose_tip_size_tooltip_form_feed' | translate }}" data-tooltip-append-to-body="true">&lt;FF&gt;</td>
+                        <td width="20%">
+                            <span data-oui-tooltip="{{ 'sms_sms_compose_tip_size_tooltip_characters_label' | translate }} {{ 'sms_sms_compose_tip_size_tooltip_form_feed' | translate }}">&lt;FF&gt;</span>
```

Comment:
```suggestion
                            <span data-oui-tooltip="{{:: 'sms_sms_compose_tip_size_tooltip_characters_label' | translate }} {{:: 'sms_sms_compose_tip_size_tooltip_form_feed' | translate }}">&lt;FF&gt;</span>
```
Does it work though ?